include/vdml/registry.h

```diff
@@ -24,6 +24,7 @@ extern "C" {
 typedef struct {
 	v5_device_e_t device_type;
 	V5_DeviceT device_info;
+	// HACK: Make pad a union of all possible types that can be stored
```

Comment:
I don't think this is necessarily a "hack"...

Reply:
I guess so; the reason I flagged it as such was because if the size of the ADI structure ever grows or if anything else that's bigger is ever stored in it, it'll have to be manually adjusted.

Reply:
Can't the compiler pad it for you?

Reply:
What do you mean?

Reply:
With the alignment attribute: https://gcc.gnu.org/onlinedocs/gcc/Common-Variable-Attributes.html#Common-Variable-Attributes

Reply:
The note has been removed, it's probably best just to leave it unchanged as at least what's in there now works and shouldn't break in the future.